src/System.Text.Encodings.Web/pkg/System.Text.Encodings.Web.pkgproj

```diff
@@ -1,9 +1,19 @@
 ﻿<Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
   <ItemGroup>
-    <ProjectReference Include="..\src\System.Text.Encodings.Web.csproj">
-      <SupportedFramework>net45;netcore45;wp8;wpa81;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+    <ProjectReference Include="..\ref\System.Text.Encodings.Web.csproj">
+      <SupportedFramework>net461;netcoreapp2.0;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
+    <ProjectReference Include="..\src\System.Text.Encodings.Web.csproj" />
+    <HarvestIncludePaths Include="lib/netstandard1.0" />
+    <!-- Since UAP and .NETCoreApp are package based we still want to enable
+     OOBing libraries that happen to overlap with their framework package.
+     This avoids us having to lock the API in our NuGet packages just 
+     to match what shipped inbox: since we can provide a new library 
+     we can update it to add API without raising the netstandard version. -->
+    <ValidatePackageSuppression Include="TreatAsOutOfBox">
```

Comment:
Did you actually need this? I don't believe that you are includding in the package the build output of netcoreapp or uap, and only includding the netstandard2.0 assembly. Was this just copy paste or was it actually intended?